test/integration/controller/core/workload_controller_test.go

```diff
@@ -171,6 +173,7 @@ var _ = ginkgo.Describe("Workload controller", func() {
 		ginkgo.AfterEach(func() {
 			gomega.Expect(framework.DeleteNamespace(ctx, k8sClient, ns)).To(gomega.Succeed())
 			gomega.Expect(framework.DeleteRuntimeClass(ctx, k8sClient, runtimeClass)).To(gomega.Succeed())
+			gomega.Expect(framework.DeleteResourceFlavor(ctx, k8sClient, flavor)).To(gomega.Succeed())
 			framework.ExpectClusterQueueToBeDeleted(ctx, k8sClient, clusterQueue, true)
```

Comment:
Maybe, we want to use `the framework.ExpectFooToBeDeleted(ctx, k8sClient, foo, true)` for all cleanup processes in all suite tests.

Probably, we can take up this in another PR.

Reply:
No needed for the reasons same as https://github.com/kubernetes-sigs/kueue/pull/415#discussion_r1046098577.